ner_v2/detectors/temporal/date/readme.md

```diff
@@ -0,0 +1,152 @@
+## Date Detector 
+
+This is the V2 version of date detector module that will detect date in multiple languages from raw text containing date entity. Currently we provide supports for 6 languages, which are
+
+- English
+- Hindi
+- Marathi
+- Gujarati
+- Telgu
+- Tamil
+
+### Usage
+
```

Comment:
Please mention api usage here as well. This is when someone wants to use it from shell.